zero/run.py

```diff
@@ -1,14 +1,14 @@
 import os
 
-from colossalai_utils.utils import init_w_col
-from common.gpt2 import gpt2_builder
-from common.train import train
-from common.utils import CONFIG, load_config, print_log
-from common.vit import vit_builder
-from deepspeed_utils.utils import init_w_ds
-from fairscale_utils.utils import init_w_fs
-from patrickstar_utils.utils import init_w_ps
-from torch_utils.utils import init_w_torch
+from zero.colossalai_utils.utils import init_w_col
+from zero.common.gpt2 import gpt2_builder

```

Comment:
did the builder really builds a bert model?

Reply:
It's zero's run.py. Just change the import path for zero.